packages/core/src/lib/Console.ts

```diff
@@ -323,8 +327,14 @@ class MemLabConsole {
     }
   }
 
-  public writeRaw(s: string): void {
-    process.stdout.write(s);
+  public writeOutput(output: string): void {
+    if (this.config.muteConfig?.muteOutput) {
+      return;
+    }
+
+    process.stdout.write(output);
+    this.log.push(output);
```

Comment:
MuteConfig is about muting console visible output, but we still want to have it in the writing log. I'm moving this statement to the beginning of the body of this method.